src/wallet/bdb.cpp

```diff
@@ -620,12 +621,12 @@ bool BerkeleyDatabase::Backup(const std::string& strDest) const
                     pathDest /= fs::PathFromString(strFile);
 
                 try {
-                    if (fs::equivalent(pathSrc, pathDest)) {
+                    if (fs::exists(pathDest) && fs::equivalent(pathSrc, pathDest)) {
```

Comment:
In commit "build: remove boost::filesystem usage" (b87f9c5edf3895df9650131fcf8551c3ad1d7301)

Note: this change was needed because fs::equivalent was throwing an exception if the path didn't exist. Earlier discussion was https://github.com/bitcoin/bitcoin/pull/20744#issuecomment-765467226